paperqa/types.py

```diff
@@ -590,19 +590,16 @@ def populate_bibtex_key_citation(  # noqa: PLR0912
                 data["bibtex"] = BibliographyData(
                     entries={data["key"]: new_entry}
                 ).to_string("bibtex")
-                # clear out the citation, since it will be regenerated
-                if data.get("overwrite_fields_from_metadata", True):
-                    data["citation"] = None
```

Comment:
Can we keep this but have `if data.get("overwrite_fields_from_metadata", True) and data.get("citation") is not None:`

I don't want to clobber this behavior in its entirety

Reply:
But if we do `data.get("citation") is not None`, either
1. The citation was None and we didn't set it to None
2. The citation had a value but we override it 
Anyway tests are probably failing because I removed that line :(